docs/content/projects/internal/CriticalPath.md

```diff
@@ -0,0 +1,43 @@
+# Critical path
+
+[Critical path](https://en.wikipedia.org/wiki/Critical_path_method) is a set of tasks that define the build duration.
+The plugin gives a report with raw data.
+
+It is very similar to what you can find in a build scan in Timeline tab:
+![build scan - critical path](https://user-images.githubusercontent.com/1104540/120473066-4b99dd00-c3af-11eb-97e0-fe9641995325.png)
+
+## Getting started
+
+Apply the plugin in the root buildscript:
+
+```kotlin
+plugins {
+    id("com.avito.android.critical-path")
+}
```

Comment:
build trace should be applied also, no?

Reply:
No, they are working independently.

This is what this PR about. Any plugin can get a critical path, it's like a regular library.
`critical-path` plugin is needed only if you need the specific report.